src/bioregistry/data/bioregistry.json

```diff
@@ -12843,6 +12843,62 @@
     "pattern": "^\\d+$",
     "uri_format": "https://europepmc.org/article/CBA/$1"
   },
+  "cbgda.disease": {
+    "contact": {
+      "email": "baojinku@scu.edu.cn",
+      "name": "Jinku Bao"
+    },
+    "contributor": {
+      "email": "m.naguthana@hotmail.com",
+      "github": "nagutm",
+      "name": "Mufaddal Naguthanawala",
+      "orcid": "0009-0009-5240-7463"
+    },
+    "description": "This collection represents diseases linked to host genes identified via genome-wide CRISPR screens. It includes detailed disease classifications, gene-disease associations, and integrated data on genetic factors contributing to disease development.",
+    "example": "Glioblastoma",
```

Comment:
We should have a discussion about if this qualifies as a semantic space or not. Combine with the fact that it's not a notable (or, from the UI design, high quality) resource, maybe we should come up with some criteria for skipping this kind of resource. This is similar to the fact that we don't import all of the Bioportal ontologies because a lot of them are junk